vasern/src/core/vasern-parser/Parser.js

```diff
@@ -253,16 +246,16 @@ const Parser = {
 
   valueTypeToStr: (dataType, value) => {
     let type = dataType;
-    if (dataType.indexOf("#") !== -1) {
+
+    if (dataType.indexOf("#") === 0) {
```

Comment:
There it still needs the case if I have an refrence with an array.

Reply:
Do you mean the data type is for example `[]#People`, where it store a list of references to `People` Collection? 

If that is the case, `valueTypeToStr` does handle it. A list of reference object will be parsed into `id_1,id_2,id_3,...`

Reply:
Yes, makes sense, than it is perfectly fine! Thanks